### PR TITLE
Richmoe/20170127

### PR DESCRIPTION
**Pull request by RichMoeMSFT**
Didn't hear much feedback yet.  Note that the Blog post is delayed one week but we want MSDN change to go out Monday.  We'll follow with a newsletter blast.  This doesn't have to be perfect on Monday but we should get it solid by the time I do the newsletter.

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C2%5C5%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201702051543357276-66%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: teams/jslibrary.md [hunk 1]
```diff
@@ -51,24 +51,6 @@ Initiates an authentication request which pops up a new windows with the specifi
...
-_Introduced in v0.4_
-
-Requests an AAD token to be issued on behalf of the app. The token is acquired from the cache if it is not expired. Otherwise a request will be sent to AAD to obtain a new token.
-
-#### Parameters
-* `authTokenRequest: `[`AuthTokenRequest`](#authentication.AuthTokenRequest)
-
-A set of values that configure the token request.
-
-### `getUser(userRequest: `[`UserRequest`](#authentication.UserRequest)`): void`
```
Comment (dfederm) [inline teams/jslibrary.md]: If we're removing these we should also remove the defined interfaces.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/deeplinks.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/richmoe/20170127/teams/deeplinks.md) | :warning:Warning | | [Details](#user-content-teams/deeplinks.md)

<a name="teams/deeplinks.md"></a>
### [teams/deeplinks.md](https://github.com/OfficeDev/Microsoft-teams-docs/blob/richmoe/20170127/teams/deeplinks.md)
 - **Line 5**: **[Warning]** Invalid file link:(~/usercontext.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C2%5C7%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201702070125505275-66%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C2%5C7%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201702070209549374-66%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**